Dockerfile

```diff
@@ -15,4 +15,10 @@ ADD . /app
 
 RUN ln -s /tmp/node_modules /app/node_modules
 
-CMD npm start
+RUN apk add openssl && \
+    mkdir -p bin && \
+    wget -qO bin/chamber https://github.com/segmentio/chamber/releases/download/v1.9.0/chamber-v1.9.0-linux-amd64 && \
```

Comment:
Should we set `chmod +x bin/chamber` here or it is not needed?

Reply:
I'm pretty sure it is! Done